ember/app/routes/admin/index.js

```diff
@@ -1,4 +1,14 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  ajax: Ember.inject.service(),
+  model() {
+    return Ember.RSVP.hash({
+      statistics: this.get('ajax').request('/admin/getStatistics', {
+        xhrFields: {
+          withCredentials: true,
+        },
+      }),
+    });
+  },
 });
```

Comment:
This is ok for now, but try to avoid ajax requests in routes. It is more efficient in controller.